src/main/java/ChatTPG.java

```diff
@@ -0,0 +1,27 @@
+import java.util.*;
+
+public class ChatTPG {
+    public static void main(String[] args) {
+        System.out.println("Hello! I'm ChatTPG");
+        System.out.println("What can I do for you?");
+        
+        Scanner scanner = new Scanner(System.in);
+        String userInput = "";
+        
+        Task task = new Task();
+
+        while (!userInput.equals("bye")) {
```

Comment:
Unnecessary check here, since you already checked for this condition a few lines down


```suggestion
        while (true) {
```